content/en/docs/concepts/cluster-administration/logging.md

```diff
@@ -133,7 +133,10 @@ Because the logging agent must run on every node, it's common to implement it as
 
 Using a node-level logging agent is the most common and encouraged approach for a Kubernetes cluster, because it creates only one agent per node, and it doesn't require any changes to the applications running on the node. However, node-level logging _only works for applications' standard output and standard error_.
 
-Kubernetes doesn't specify a logging agent, but two optional logging agents are packaged with the Kubernetes release: [Stackdriver Logging](/docs/tasks/debug-application-cluster/logging-stackdriver/) for use with Google Cloud Platform, and [Elasticsearch](/docs/tasks/debug-application-cluster/logging-elasticsearch-kibana/). You can find more information and instructions in the dedicated documents. Both use [fluentd](https://www.fluentd.org/) with custom configuration as an agent on the node.
+Kubernetes doesn't specify a logging agent, but two optional logging agents are integrated with
+Kubernetes: [Stackdriver Logging](https://cloud.google.com/stackdriver/docs) and
+[Elasticsearch](https://www.elastic.co/elastic-stack). Both logging agents use
+[fluentd](https://www.fluentd.org/) with custom configuration as an agent on the node.
```

Comment:
I think this is dated. By now, several more products work with Kubernetes for logging. It's a long list.

Reply:
Yes, I agree. I could continue to clean up in this page.